assets/queries/openAPI/components_object_fixed_field_key_improperly_named/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	obj := doc.components[field][key]
+	regex.match(`^[a-zA-Z0-9\\.\\-_]+$`, key) == false
```

Comment:
Can we move this to a function `is_alphanumeric()`

Reply:
Of course 😄